Quantum-Control-Applications/ESR/README.md

```diff
@@ -0,0 +1,18 @@
+# ESR experiments on an ensemble
+
+## Basic Files
+These files showcase various ESR experiments that can be done on an ensemble.
```

Comment:
I would specify what ESR means (Electron Spin Resonance), I didn't know the acronym...
And also specify the setup, is it for SC (fixed or tunable freq), QD, NV?